Description: Fix desktop file as Freedesktop.org's per spec.
Author: Chow Loong Jin <hyperair@gmail.com>
Author: James Cowgill <james410@cowgill.org.uk>
Forwarded: http://forums.codelite.org/viewtopic.php?f=13&t=2506
Applied-Upstream: https://github.com/eranif/codelite/commit/13fc6dfdedb28b3f5ee1df335bd40af6ee31f6d5
Last-Update: 2014-05-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Runtime/codelite.desktop.template
+++ b/Runtime/codelite.desktop.template
@@ -1,9 +1,9 @@
 [Desktop Entry]
-Encoding=UTF-8
-Name=codelite
+Name=CodeLite
 Exec=codelite %f
 Icon=%%PREFIX%%/share/codelite/images/cubes.png
 Terminal=false
 Type=Application
 Categories=Development;
 StartupNotify=true
+Keywords=IDE;development;programming;
